inc/namespace.php

```diff
@@ -54,3 +56,30 @@ function load_publication_checklist() {
 
 	require_once Altis\ROOT_DIR . '/vendor/humanmade/publication-checklist/plugin.php';
 }
+
+/**
+ * Load the QM Collector for debugging notifications.
+ */
+function load_query_monitor_debug() {
```

Comment:
For this function there's quite a clean approach to this already in the dev tools module here:

https://github.com/humanmade/altis-dev-tools/blob/bf330a5912ae0baec7cac1fc7029ce83e723de11/inc/namespace.php#L44-L79

The key change is to add a classmap entry to the `autoload` section in the `composer.json` file like so: 

https://github.com/humanmade/altis-dev-tools/blob/191fdb59c961b81b7825304f4d96dfba5cedae2a/composer.json#L11-L18

Means you can just rely on composer's autoloader then.

Reply:
ok, I added the `classmap` but now I get `Uncaught Error: Class 'Altis\Workflow\Notifications\QM_Collector_Notifications' not found in /usr/src/app/packages/workflow/inc/namespace.php on line 72`
I also added `"johnbillion/query-monitor": "^3.5.2"` to the composer require?
and then I ran `composer update`

Reply:
I think this is because the `autoload_classmap` file doesn't contain those classes. I tried `composer dumpautoload` but that didn't fix it